### PR TITLE
Remove the unnecessary header text in the organization section.

### PR DESCRIPTION
**Pull request by Achintha444**
### Purpose
Remove the unnecessary header text in the organization section.

#### Sub organization
Screenshot 2024-07-22 at 15 42 38

#### Root organization
Screenshot 2024-07-22 at 15 34 44

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.organizations.v1/components/organization-list.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    margin-left: 0 !important;
+}
+
+.organization-active-icon-active {
+    color: $organization-active-icon-active-color
+}
+
+.organization-active-icon-inactive {
+    color: $organization-active-icon-inactive-color
+}
```
Comment (pavinduLakshan) [inline features/admin.organizations.v1/components/organization-list.scss:33]: Can we nest the css?

eg: 

.organization-status-icon {
    .active {
     
     }

     .inactive {
     
     }
}

and then we can use `organization-status-icon active` etc

File: features/admin.organizations.v1/components/organization-list.tsx [hunk 2]
```diff
@@ -325,10 +325,13 @@ export const OrganizationList: FunctionComponent<OrganizationListPropsInterface>
                                     trigger={
                                         (<Icon
                                             data-componentid={ `${ componentId }-org-status-icon` }
-                                            className="mr-2 ml-0 vertical-aligned-baseline"
+                                            className={ "organization-active-icon "
+                                                + (organization.status === "ACTIVE"
+                                                    ? "organization-active-icon-active"
+                                                    : "organization-active-icon-inactive")
+                                            }
```
Comment (pavinduLakshan) [inline features/admin.organizations.v1/components/organization-list.tsx:332]: Let's use `classNames` package for this.

File: features/admin.organizations.v1/components/organization-list.scss (hunk 1 above)
Comment (Achintha444) [reply features/admin.organizations.v1/components/organization-list.scss:33]: ack that will be better will do that

Comment (Achintha444) [reply features/admin.organizations.v1/components/organization-list.scss]: Fixed with https://github.com/wso2/identity-apps/pull/6640/commits/b34467767ee4bbac6d8f95413ede92091a077118

File: features/admin.organizations.v1/components/organization-list.tsx (hunk 2 above)
Comment (Achintha444) [reply features/admin.organizations.v1/components/organization-list.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/6640/commits/b34467767ee4bbac6d8f95413ede92091a077118

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**